File: U1/TI1.3 RESTFULL_JAVA_SINBDD_GR01/02 CLIWEB/ConUni_Rest_Web_GR01/src/main/java/ec/edu/monster/model/ConversionResult.java
package ec.edu.monster.model;

/**
 * Modelo para el resultado de una conversión REST
 * @author YourName
 */
public class ConversionResult {
    private boolean exitoso;
    private ResultadoConversion resultado;
    private ErrorConversion error;
    
    // Getters y Setters
    public boolean isExitoso() {
        return exitoso;
    }
    
    public void setExitoso(boolean exitoso) {
        this.exitoso = exitoso;
    }
    
    public ResultadoConversion getResultado() {
        return resultado;
    }
    
    public void setResultado(ResultadoConversion resultado) {
        this.resultado = resultado;
    }
    
    public ErrorConversion getError() {
        return error;
    }
    
    public void setError(ErrorConversion error) {
        this.error = error;
    }
}

File: U1/TI1.4 RESTFULL_DOTNET_SINBDD_GR01/02CLIWEB/ConUni_CLIWEB_Rest/ec/edu/monster/models/ConversionResult.cs
using System;

namespace ConUni_CLIWEB_Rest.ec.edu.monster.models
{
    /// <summary>
    /// Modelo para el resultado de una conversión REST
    /// </summary>
    public class ConversionResult
    {
        public bool Exitoso { get; set; }
        public ResultadoConversion Resultado { get; set; }
        public ErrorConversion Error { get; set; }

        public ConversionResult()
        {
            Exitoso = false;
            Resultado = null;
            Error = null;
        }
    }
}